test/BUILD

```diff
@@ -216,3 +216,30 @@ scala_binary(
     srcs = ["src/main/scala/scala/test/only_java/Alpha.java"],
     main_class = "scala.test.Alpha",
 )
+
+#Import native libs
+# Native library created on OSX with
+#    gcc -shared -o libNativeLibCompiledElsewhere.dylib NativeLib.c
+cc_library(
+    name = "NativeLibCompiledElsewhere",
+    srcs = [
+        "libNativeLibCompiledElsewhere.dylib",
+    ],
+)
+
+java_test(
+    name = "NativeTestJava",
+    srcs = ["NativeTestJava.java"],
+    test_class = "scala.test.NativeTestJava",
+    deps = [
+        ":NativeLibCompiledElsewhere",
+    ],
+)
+
+scala_test(
+    name = "NativeTestScala",
+    srcs = ["NativeTestScala.scala"],
+    deps = [
+        ":NativeLibCompiledElsewhere",
```

Comment:
I think this should be a `data`

Reply:
I have the same issue, and puting the `cc_library` target as `data` do not solve it.
I don't know if it changes anything, but in my case the `so` is embedded in third party jar. When using it in a `scala_binary` it works fine, but from within a `scala_test` it does not appear in `java.library.path`.